evaluation.py

```diff
@@ -54,17 +55,22 @@ def _addAbsoluteFile(self,file,flist):
             raise ValueError("File '"+file+"' not found.")
         flist.append(file)
 
-    def addData(self,file,location="auto"):
+    def addData(self,file,location="auto",destination=None):
         """
         Adds a "data" file to the run, an immutable dependency of the process.
 
         Parameters
         ----------
-        file     : Path to the file.
-        location : Type of path, "relative" (to the parent of "dir"), "absolute" (the path
-                   is immediately converted to an absolute path, the file must exist),
-                   or "auto" (tries "absolute" first, reverts to "relative"). 
+        file       : Path to the file.
+        location   : Type of path, "relative" (to the parent of "dir"), "absolute" (the path
+                     is immediately converted to an absolute path, the file must exist),
+                     or "auto" (tries "absolute" first, reverts to "relative").
+        destination: Filename to be set at the destination. Discard any additional file path.
+                     Default destination is the regular filename
         """
+        if destination is None: destination = os.path.basename(file)
+        self._dataFilesDestination.append(destination)
+
```

Comment:
```suggestion
        if destination is None: destination = file
        self._dataFilesDestination.append(os.path.basename(destination))

```

Reply:
👍 makes sense to have that logic in one place